src/client/pfs.go

```diff
@@ -181,14 +181,19 @@ func (c APIClient) StartCommitParent(repoName string, branchName string, parentB
 // FinishCommit ends the process of committing data to a Repo and persists the
 // Commit. Once a Commit is finished the data becomes immutable and future
 // attempts to write to it with PutFile will error.
-func (c APIClient) FinishCommit(repoName string, branchName string, commitID string) error {
+func (c APIClient) FinishCommit(repoName string, branchName string, commitID string) (retErr error) {
+	defer func() { retErr = grpcutil.ScrubGRPC(retErr) }()
```

Comment:
ScrubGRPC prevents inspecting structured errors.  I think we should stop using it.

Reply:
Sure, but we should do that separate from this PR.